packages/story-editor/src/elements/text/karma/edit.karma.js

```diff
@@ -176,8 +177,14 @@ describe('TextEdit integration', () => {
     describe('edit mode', () => {
       it('should not change height when entering and exiting edit mode', async () => {
         await fixture.events.click(fixture.editor.library.textAdd);
-        await waitFor(() => fixture.editor.canvas.framesLayer.frames[1].node);
-        frame = fixture.editor.canvas.framesLayer.frames[1].node;
+        await waitFor(() => {
+          const node = fixture.editor.canvas.framesLayer.frames[1].node;
+          if (!node) {
+            throw new Error('node not ready');
+          }
+          expect(node).toBeTruthy();
```

Comment:
Is there still a purpose for adding `expect(node).toBeTruthy();` if it would throw an error above in case of `!node`? Seeing this in some other places as well. Thoughts?

Reply:
No we could just throw and that would suffice.

Reply:
i don't mind this despite the redundancy in some places, both patterns (like you said, Brooke) work just fine, i think this is a good reminder of how to actually use `waitFor` and having it so easy to read sometimes is helpful for future versions of ourselves.